src/routes/tools/extreme-precipitation/index.svelte

```diff
@@ -192,7 +199,7 @@
         ...potParams,
         ...(pct && { pct }),
       });
-      thresholdStore.set(thresh);
+      thresholdStore.set(+format(`.${DEFAULT_THRESHOLD_PRECISION}f`)(thresh));
```

Comment:
reformats threshold value to 2 decimal places

Reply:
@mukhtyar you might want to extract that code into a separate local function as it's being used in multiple places.

Reply:
fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/177/commits/b93d4e1575fc1f8f7fb51ac68be685a911aa6bda